### ISSUE TITLE
typo in competitive pages: "Wite skew"

### ISSUE THREAD

**Issue by ajwerner** [cockroachdb/docs#6864]
See here:

https://github.com/cockroachdb/docs/blob/master/v19.2/cockroachdb-in-comparison.md

Also of note is that the capitalization is inconsistent.

**Comment by aniketrb-github** [comment]
Hi @Amruta-Ranade, can you assign me this issue.
From the description, I understand we need to fix the typos and the unnecessary inconsistent capitalization.

I just rectified the [above-mentioned file](https://github.com/cockroachdb/docs/blob/master/v19.2/cockroachdb-in-comparison.md), and yes I did see the **inconsistent capitalization**, but there are no misspells for Write Skew in the same file.
So, just wanted to understand like should I look into all the **cockroachdb-in-comparison.md files** of all versions that are maintained here? like for instance [v19.1]( https://github.com/cockroachdb/docs/tree/master/v19.1), [v1.0 ](https://github.com/cockroachdb/docs/tree/master/v1.0), v19.2, v2.0, v2.1, v20.1 etc. 

It would be great if there are more details regarding the target area to be fixed.
Can you please guide me a little here? 

Thanks,
Aniket Bharsakale.

**Comment by Amruta-Ranade** [comment]
@aniketrb-github Hmm..someone might have fixed the typo as part of another docs PR and forgot to close this issue. However, I think the inconsistent capitalization is still an issue. Can you open a PR fixing the inconsistent capitalization? Once you get the :looks good to me: on the content, you can open a follow-up PR to port the capitalization changes to 20.1 and 20.2 docs as well

**Comment by aniketrb-github** [comment]
Can you assign me this [5330 issue](https://github.com/cockroachdb/docs/issues/5330) first please, since this looks like a quick fix.
Once this is done I can check on [contribution list issue](https://github.com/cockroachdb/docs/issues/6864) instead.
Is it fine?

**Comment by Amruta-Ranade** [comment]
Done!

**Comment by aniketrb-github** [comment]
@Amruta-Ranade Please can I work on this issue...since no one is in the queue? :)

**Comment by Amruta-Ranade** [comment]
Go for it :)

**Comment by aniketrb-github** [comment]
> Go for it :)

Thanks.

**Comment by Amruta-Ranade** [comment]
Can you open a PR for the inconsistent capitalization issue for 19.1 and let me take a look at it before you work on 20.1 and 20.2 versions?

**Comment by aniketrb-github** [comment]
> Can you open a PR for the inconsistent capitalization issue for 19.1 and let me take a look at it before you work on 20.1 and 20.2 versions?

Oh Okay, Do you mean to say I should work on v19.1 first, and based on that I'll get a go-ahead for other versions, right?

**Comment by Amruta-Ranade** [comment]
Yep

**Comment by aniketrb-github** [comment]
> Yep

Sure. Thanks for the suggestion. I'm on it.

**Comment by Amruta-Ranade** [comment]
@aniketrb-github Heads-up: The Hacktoberfest organizers are changing the rules to combat spam. To have your PR count for Hacktoberfest, hold off on opening the PR until at least Tuesday to give us time to implement the new rules. All the PRs you have already opened will be counted :)

**Comment by aniketrb-github** [comment]
> @aniketrb-github Heads-up: The Hacktoberfest organizers are changing the rules to combat spam. To have your PR count for Hacktoberfest, hold off on opening the PR until at least Tuesday to give us time to implement the new rules. All the PRs you have already opened will be counted :)

Oh...well, I didn't know that. Thanks for the heads-up though. :)

**Comment by aniketrb-github** [comment]
found this **(v20.1)** [CockroachDB in Comparison page](https://www.cockroachlabs.com/docs/v20.1/cockroachdb-in-comparison.html), deployed actual page link with which I can now compare with.

I was trying to update the [HTML source code](https://github.com/cockroachdb/docs/blob/master/v19.2/cockroachdb-in-comparison.md) on the browser in the GitHub editing window itself for a few minutes without having anything to compare my work/updates with and I was just not able to derive any sense out of it. Lol. 😄 

IMO, both links(deployed page link & the source code page link should be provided by the reporter which makes it a bit easy.

**Comment by aniketrb-github** [comment]
Just a question here @Amruta-Ranade 
Previously when I did my commits there were the automated checks that got executed after each commit which certainly is the CI/CD implementation, I understand, if I'm not wrong.
So, my question was on which environment or version are my changes visible as soon as we commit? (the deployed link)
Is it available to us or can you let me know about it?

**Comment by aniketrb-github** [comment]
> Just a question here @Amruta-Ranade
> Previously when I did my commits there were the automated checks that got executed after each commit which certainly is the CI/CD implementation, I understand, if I'm not wrong.
> So, my question was on which environment or version are my changes visible as soon as we commit? (the deployed link)
> Is it available to us or can you let me know about it?

I guess the checks happen only when we raise a PR and not an individual/several commit(s).

**Comment by Amruta-Ranade** [comment]
As I understand it, the checks happen for every commit after you open a PR, but you need to open a PR first. I might be wrong though -- let me check with my colleague next Tuesday and confirm it for you. (Tuesday bc w have a company holiday on Monday and I don't want to disturb them over the long weekend :) )

**Comment by Amruta-Ranade** [comment]
The contributing guide (https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md) has instructions on "Use Jekyll to build a version of the site locally so you can view your changes in a browser"

**Comment by aniketrb-github** [comment]
> As I understand it, the checks happen for every commit after you open a PR, but you need to open a PR first. I might be wrong though -- let me check with my colleague next Tuesday and confirm it for you. (Tuesday bc w have a company holiday on Monday and I don't want to disturb them over the long weekend :) )

sure sure, not a problem.

**Comment by Amruta-Ranade** [comment]
Addressed in https://github.com/cockroachdb/docs/pull/8561
